package.json

```diff
@@ -22,33 +22,34 @@
     "npm": "3.10.8"
   },
   "dependencies": {
-    "bunyan": "^1.8.1",
-    "console-stamp" : "latest",
-    "express": "latest",
-    "express-enforces-ssl":"latest",
-    "express-helpers": "latest",
-    "express-session":"latest",
-    "body-parser":"latest",
-    "ejs": "2.4.2",
-    "helmet": "latest",
-    "pg" : "6.1.0",
-    "pg-large-object": "latest",
     "async": "latest",
+    "body-parser": "latest",
+    "bunyan": "^1.8.1",
+    "chai": "latest",
     "chart.js": "^2.1.3",
+    "connect-flash": "0.1.1",
+    "console-stamp": "latest",
     "crypto": "latest",
+    "db-migrate": "0.10.0-beta.12",
+    "db-migrate-pg": "latest",
     "dotenv": "latest",
-    "multer": "latest",
+    "ejs": "2.4.2",
+    "express": "latest",
+    "express-enforces-ssl": "latest",
+    "express-helpers": "latest",
+    "express-session": "latest",
+    "express-validator": "latest",
+    "helmet": "latest",
     "mocha": "latest",
-    "sinon": "latest",
-    "chai": "latest",
+    "multer": "latest",
     "passport": "0.3.2",
     "passport-azure-ad": "2.0.2",
     "passport-local": "1.0.0",
+    "pg": "6.1.0",
+    "pg-large-object": "latest",
     "sanitize-html": "latest",
-    "db-migrate": "0.10.0-beta.12",
-    "db-migrate-pg": "latest",
-    "express-validator" : "latest",
-    "connect-flash" : "0.1.1"
+    "sinon": "latest",
+    "underscore": "^1.8.3"
```

Comment:
FYI: I only added the underscore dependency. The remaining changes were made by npm auto-sorting.